views/game.erb

```diff
@@ -0,0 +1,27 @@
+<header>
+<link rel ="stylesheet" type="text/css" href="/style.css"/>
+</header>
+
+<body>
+<br>
+<%= @player1.name %> plays against <%= @player2.name %>
+<p>------------------------------------------------------------</p>
+<br>
+
```

Comment:
Could you find a way of allowing the users to choose weapons, using the same form, without seeing each other's input? :)